src/Channel.js

```diff
@@ -5,19 +5,27 @@ const ChannelAudienceWorld = require('./ChannelAudience/World');
 const ChannelAudiencePrivate = require('./ChannelAudience/Private');
 const util = require('util');
 
+class ChannelError extends Error {}
```

Comment:
Unless we're exporting `ChannelError` so people can catch and handle ChannelErrors specifically there's no real reason to actually use this instead of a plain `Error`